test/functional/ui/input_spec.lua

```diff
@@ -126,3 +128,208 @@ describe('input utf sequences that contain CSI/K_SPECIAL', function()
     expect('…')
   end)
 end)
+
+
+-- Testing 'keymap'.
```

Comment:
Let's put this in `test/functional/options/keymap_spec.lua`. 

(`test/functional/ui/...` is for testing the UI)